src/bouquet.c

```diff
@@ -830,7 +830,7 @@ static char *
 bouquet_class_services_rend ( void *obj, const char *lang )
 {
   bouquet_t *bq = obj;
-  const char *sc = N_("Services Count %zi");
+  const char *sc = N_("Services count %zi");
```

Comment:
"Services" or "service"? I suspect the latter - like "apple cart", "car park" or "head count"?